packages/backend/src/app/database/migration/1677286751592-billing.ts

```diff
@@ -20,7 +20,7 @@ export class billing1677286751592 implements MigrationInterface {
         console.log("rolling back migration billing1677286751592");
         await queryRunner.query(`ALTER TABLE "project_usage" DROP CONSTRAINT "fk_project_usage_project_id"`);
         await queryRunner.query(`ALTER TABLE "project_plan" DROP CONSTRAINT "fk_project_plan_project_id"`);
-        await queryRunner.query(`DROP INDEX "public"."idx_project_usage_project_id"`);
+        await queryRunner.query(`DROP UNIQUE INDEX "public"."idx_project_usage_project_id_nextResetDatetime"`);
```

Comment:
I think we should avoid editing old migrations, even if edits are trivial, it would be better to create a new migration file.